doc/api/url.md

```diff
@@ -440,8 +440,8 @@ console.log(u.href);
 // fish://example.org
 ```
 
-The protocol schemes considered to be special by the WHATWG URL Standard
-include: `ftp`, `file`, `gopher`, `http`, `https`, `ws`, and `wss`.
+According to the WHATWG URL Standard, special protocol schemes include `ftp`,
```

Comment:
Instead of “include” we could just say “are”.

Reply:
I'm good with either, but I imagine "include" was originally chosen to future-proof the doc in case additional "special" schemes are added at a later date.

Reply:
I don't think that's very probable; in fact it's more probable that `gopher` be removed (https://github.com/whatwg/url/issues/342).

Reply:
Hot dang! Not gopher!!!!

Ok, you convinced me! I'll change it.